forms/models.py

```diff
@@ -133,7 +135,8 @@ class RadioSheet(SheetModel):
     comments = field_comments(_('(N/A) Describe any photographs included in the story and the conclusions you draw from them.'))
 
     def __str__(self):
-        return f"{self.channel} {str(self.start_time)} story {str(self.item_number)}"
+        item_number = f"story {str(self.item_number)}" if self.item_number else ""
+        return f"{self.channel} {str(self.start_time)} {item_number}"
```

Comment:
Nit... Make the space between `start_time` and `item_number` part of `item_number` so that if there is no `item_number`, there is no space.